src/app/components/common/modal/modal.component.css

```diff
@@ -0,0 +1,110 @@
+﻿/* MODAL STYLES
+-------------------------------*/
+
+.jw-modal {
+    /* modal container fixed across whole screen */
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+
+    /* z-index must be higher than .jw-modal-background */
+    z-index: -10;
+        
+    /* enables scrolling for tall modals */
+    overflow: auto;
+}
+
+.jw-modal.open {
+    z-index: 1000;
+}
+
+.slide-in-form {
+    width: 400px;
+    position: fixed;
+    top: 50px;
```

Comment:
this one should be `top: 48px` since the height of top menu bar is 48px. otherwise, there will be a white line between it and the side-form (also the black mask, and second level side menu) in light theme.